be/src/storage/delta_writer.cpp

```diff
@@ -389,6 +389,19 @@ void DeltaWriter::_set_state(State state, const Status& st) {
     }
 }
 
+Status DeltaWriter::_check_op_cols() {
+    // If more than one column has __op in PK table, then return error
+    // because PK table can't use __op as column name
+    if (_tablet->updates() != nullptr && _opt.slots != nullptr) {
+        int op_count = std::count_if(_opt.slots->begin(), _opt.slots->end(),
+                                     [](const auto& slot) { return slot != nullptr && slot->col_name() == "__op"; });
```

Comment:
It might be safer to check if the only one `__op` column is the last column.

Reply:
After talk with Yanbo, I remove check on BE side.